src/app/app.module.ts

```diff
@@ -51,8 +51,8 @@ const PREFERRED_DATE_FORMATS = {
     BrowserAnimationsModule,
     DashboardModule,
     MatMomentDateModule,
-    StoreModule.forRoot(reducers, { metaReducers }),
-    StoreModule.forRoot({ router: routerReducer }),
```

Comment:
_sigh_ seems there's some kind of mutation of the state somewhere and you can't load up the campaign feature (campaign/flight form) without running into a "Cannot freeze" error. These runtime checks get turned on by default in ngrx 9.

Reply:
Probably yes because it could potentially lead to weird bugs. It could be hard to track down, but there are freeze utilities you might be able to apply to individual reducers to find where there is mutation happening. My guess is that it's in the flight reducer since that's a deep object, but I didn't spend enough time tracking it down. I did try the ngrx-store-freeze application wide, and I got the same error.